StreamChatClient_v3/APIClient/Endpoints/MemberEndpoints.swift

```diff
@@ -6,7 +6,7 @@
 import Foundation
 
 struct MemberPayload<ExtraData: UserExtraData>: Decodable {
-    let roleRawValue: String
+    let roleRawValue: String?
```

Comment:
It would be great to add a comment here to know why it's optional and how it should a member should react without the role.